packages/commerce-sdk-react/src/utils.ts

```diff
@@ -61,6 +61,17 @@ export const getCookieSameSiteAttribute = () => {
     return !isLocalHost && isOriginTrusted(parentOrigin) ? 'none' : 'Lax'
 }
 
+/**
+ * Gets the value to use for the `sameSite` cookie attribute for cookies that need to work with external APIs.
+ * @returns `"none"` for cookies that need to be sent to external APIs, otherwise uses the default logic
+ */
+export const getExternalApiCookieSameSiteAttribute = () => {
```

Comment:
Just wondering if we really need this extra encapsulation or can we conditionally set the value directly in the `getExternalApiCookieAttributes` function defined below similar to how we do it for `secure: true`